### PR TITLE
male to female ratio

### PR DESCRIPTION
**Pull request by ngoldstein51**
Finds the ration of males to females in the file

### CODE REVIEW

File: verifier.py [hunk 1]
```diff
@@ -298,6 +298,21 @@ def US36_verify_death_at_recent_30_days(individual):
...
+# US46: Count the percentage of males and females
+def US46_male_female_ratio(individualsDict=individualsDict):
+    male_num = 0
+    female_num = 0
+    for id, individual in individualsDict.items():
+        name = individual["name"]
+        birthday = individual["birthday"]
+
+        if individual['gender'] == 'M':
+            male_num += 1
```
Comment (DQSevilla) [inline verifier.py:363]: Call this

**Review by DQSevilla — APPROVED** [review]

**Review by DQSevilla — CHANGES_REQUESTED** [review]
fix

File: verifier.py [hunk 2]
```diff
@@ -298,6 +298,21 @@ def US36_verify_death_at_recent_30_days(individual):
...
+        else:
+            female_num += 1
+
+    return male_num/(male_num+female_num), female_num/(male_num+female_num)
```
Comment (DQSevilla) [inline verifier.py]: Pfemale = 1 - Pmale, don't calculate it

Comment (DQSevilla) [reply verifier.py]: Also either return as a percent (* 100) or print as a percent when you actually call it

File: verifier.py (hunk 1 above)
Comment (ngoldstein51) [reply verifier.py:363]: What does that mean?

Comment (ngoldstein51) [reply verifier.py:363]: Oh, nevermind

**Review by ngoldstein51 — COMMENTED** [review]
Changes made

**Review by Jarbo52 — APPROVED** [review]
lgtm

**Review by DQSevilla — APPROVED** [review]

**Pull request merged**